### ISSUE TITLE
Having automatic device status detection at on is causing lag / hiccups on logitech mx master 3s 

### ISSUE THREAD

**Issue by Jeanseb13** [csutorasa/XOutput#765]
Hi there I really like your software, I've been trying for weeks to find the issue of my mouse mx master 3s connected with bluetooth or logitech dongle. I updated everything until I close Xoutput by error then I realize the bug of my mouse having lag hiccups every 5 or 6 seconds is gone ! Then I tried to disable automatic device status detection and the bug is gone. I dont know if its a fix you could adress but thank you anyway for a very good software.

**Comment by AlexDeFoc** [comment]
yeah I just disable it and everything works. Now when i put my controller in the pc and wanna use xoutput I click Force refresh and done.

**Comment by AlexDeFoc** [comment]
if you want a why its because a function its callef every few seconds i think and when that happens your mouse and mine too would just hiccup and thats the thing. But when you dont have auto detection (so the app searches for all fevices connected like asking a neighbour if he is alive he gets irritated)

**Comment by Designbykivi** [comment]
Had the same issue on Logitech Lift mouse, both with the dongle and bluetooth. Disabled Automatic Device Status Detection and the stutter every 5 seconds is gone.
Also happens with a Wacom Intuos Pro.
